packages/components/tests/acceptance/components/hds/link/standalone-test.js

```diff
@@ -0,0 +1,20 @@
+import { module, test } from 'qunit';
+import { visit, currentURL } from '@ember/test-helpers';
+import { setupApplicationTest } from 'dummy/tests/helpers';
+import { a11yAudit } from 'ember-a11y-testing/test-support';
+
+module('Acceptance | Component | hds/link/standalone', function (hooks) {
+  setupApplicationTest(hooks);
+
+  test('visiting /components/link/standalone', async function (assert) {
+    await visit('/components/link/standalone');
+
+    assert.strictEqual(currentURL(), '/components/link/standalone');
+  });
```

Comment:
Same here, we can drop this test before merging.

Reply:
dropped 👍 thanks for catching that